src/interfaces/assistants_web/src/components/Shared/Textarea/Textarea.tsx

```diff
@@ -6,26 +6,37 @@ import { forwardRef } from 'react';
 import { STYLE_LEVEL_TO_CLASSES, Text } from '@/components/Shared/Text';
 import { cn } from '@/utils';
 
+import { InputLabel } from '../InputLabel';
+
 type Props = Omit<React.HTMLProps<HTMLTextAreaElement>, 'onChange' | 'value'> & {
   value?: string;
   label?: string;
+  labelTooltip?: React.ReactNode;
+  placeholder?: string;
```

Comment:
nit: placeholder and className are already defined into `React.HTMLProps<HTMLTextAreaElement>`  so there is no need to add them, unless you wanted them to always be defined